packages/ariakit/src/heading/__examples__/heading/index.tsx

```diff
@@ -0,0 +1,21 @@
+import { Heading, HeadingLevel } from "ariakit/heading";
+import "./style.css";
+
+export default function Example() {
+  return (
+    <HeadingLevel>
+      <div role="heading-container" className="headingContainer">
+        <Heading>Lacus Et Semper Turpis Massa Commodo Cum</Heading>
+        <p>Torquent penatibus ipsum nascetur cursus primis lobortis</p>
+        <HeadingLevel>
+          <Heading>Ac Nullam</Heading>
+          <p>Volutpat metus id purus dignissim fusce Tellus egestas.</p>
+        </HeadingLevel>
+        <HeadingLevel>
+          <Heading>Quis Placerat</Heading>
+          <p>Platea justo lectus. Praesent. Et sodales pellentesque</p>
+        </HeadingLevel>
+      </div>
+    </HeadingLevel>
```

Comment:
Nice! `heading-container` is not a valid role though.
```suggestion
    <div className="wrapper">
      <HeadingLevel>
        <Heading>Lacus Et Semper Turpis Massa Commodo Cum</Heading>
        <p>Torquent penatibus ipsum nascetur cursus primis lobortis</p>
        <HeadingLevel>
          <Heading>Ac Nullam</Heading>
          <p>Volutpat metus id purus dignissim fusce Tellus egestas.</p>
        </HeadingLevel>
        <HeadingLevel>
          <Heading>Quis Placerat</Heading>
          <p>Platea justo lectus. Praesent. Et sodales pellentesque</p>
        </HeadingLevel>
      </HeadingLevel>
    </div>
```